src/configs/redisConfigs.js

```diff
@@ -0,0 +1,26 @@
+import redis from 'redis';
+import dotenv from 'dotenv';
+import pino from 'pino';
+
+dotenv.config();
+
+const consoleLog = pino();
+const clientConnection = redis.createClient({
+  url: process.env.REDIS_URL,
+});
+
+clientConnection.on('connect', (err) => {
+  if (err) {
```

Comment:
@MCFrank16 I think because we already know that the connection can pass, we don't need these logs